scripts/tests/provisioning/test_safemode.expect

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/expect -f
```

Comment:
Can you rename the test to something more descriptive like, test_migration_to_safemode?